charts/logzio-logs-collector/README.md

```diff
@@ -0,0 +1,134 @@
+| tolerations              | Tolerations for pod assignment.                                                  | `[]`                                     |
+| affinity                 | Affinity rules for pod assignment.                                               | Complex structure (see `values.yaml`)    |
+| priorityClassName        | Scheduler priority class name.                                                   | `""`                                     |
+| extraEnvs                | Extra environment variables to set in the pods.                                  | `[]`                                     |
+| ports                    | Defines ports configurations.                                                    | Complex structure (see `values.yaml`)    |
+| resources                | CPU/memory resource requests/limits.                                             | `limits.cpu:250m` `limits.cpu:512Mi`     |
+| podAnnotations           | Annotations to add to the pod.                                                   | `{}`                                     |
+| podLabels                | Labels to add to the pod.                                                        | `{}`                                     |
+| hostNetwork              | Use the host's network namespace.                                                | `false`                                  |
+| dnsPolicy                | Pod DNS policy.                                                                  | `""`                                     |
+| livenessProbe            | Liveness probe configuration.                                                    | (see `values.yaml`)                      |
+| readinessProbe           | Readiness probe configuration.                                                   | (see `values.yaml`)                      |     
+| service.enabled          | Enable the creation of a Service.                                                | `true`                                   |
+| ingress.enabled          | Enable ingress resource creation.                                                | `false`                                  |
+| podMonitor.enabled       | Enable the creation of a PodMonitor.                                             | `false`                                  |
+| networkPolicy.enabled    | Enable NetworkPolicy creation.                                                   | `false`                                  |
+| useGOMEMLIMIT            | Set GOMEMLIMIT env var to a percentage of resources.limits.memory.               | `false`                                 |
+
+### Configure customization options
+
+You can use the following options to update the Helm chart parameters: 
+
+* Specify parameters using the `--set key=value[,key=value]` argument to `helm install`
+
+* Edit the `values.yaml`
+
+* Overide default values with your own `my_values.yaml` and apply it in the `helm install` command. 
+
```

Comment:
Add multiline explanation (as in our FluentD docs)[https://github.com/logzio/logzio-helm/tree/master/charts/fluentd#configuring-fluentd-to-concatenate-multiline-logs-using-a-plugin], common use case and seems to happen when testing.

Reply:
Added `multiline.md` guide